src/display_context.hpp

```diff
@@ -57,6 +57,8 @@ class display_context {
 	virtual const std::vector<std::string> & hidden_label_categories() const = 0;
 	std::vector<std::string> & hidden_label_categories_ref();
 	const team& get_team(int side) const;
+
+	template<typename T = void>
```

Comment:
in display_context.hpp the class team is not a complete type and std::vector<team>::size() requires team to be complete. The intention  of that line is to make sure that it is only complied when it's actuall used (called) which which case team.hpp is also included.